db/cache.js

```diff
@@ -8,11 +8,11 @@ export default () => {
     log.info(`caching promises for ${users.length} users`)
     users.forEach(user => {
       user.getValidPromises().then(promises => {
-        const score = calculateReliability(promises)
-        const doneOrDueCount = promisesIncluded(promises)
-        log.debug(`caching ${user.username}'s score:`, score, doneOrDueCount)
-        user.update({ score })
-        user.update({ doneOrDueCount })
+        const rel = calculateReliability(promises)
```

Comment:
you can make this even shorter with es6 destructuring 😄 :
```js
	const { score, counted } = calculateReliability(promises)
	log.debug(`caching ${user.username}'s score:`, score, counted)
	user.update({ score, counted })
```

Reply:
Will do - the depths of my ignorance of es6 cunningness are showing through ;-)